webapp/store/reducers/app.js

```diff
@@ -3,7 +3,8 @@ import { SET_WINDOW } from '../constants/app';
 const initialState = {
   port: null,
   protocol: null,
-  ssl: null
+  ssl: null,
+  socketPort: 8000
```

Comment:
I think it would be ideal to define the socket port in just 1 place and pass that value through to webpack and have webpack set the value here and the pass through the value to the backend, but I don't think its that huge of a deal that we aren't doing it that way now since its expected to change again

Reply:
I agree with this. I'll add in a `SOCKET_PORTS` to webpack and set `BSOCK_PORT` if none is set on `process.env`

Reply:
Updated!